automation/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/rstudio/RStudioPage.scala

```diff
@@ -45,7 +45,7 @@ class RStudioPage(override val url: String)(implicit override val authToken: Aut
   def withRShinyExample[T](exampleName: String)(testCode: RShinyPage => T): T = {
     // Enter commands to launch the shiny app
     switchToNewTab {
-      val launchCommand = s"""shiny::runExample("$exampleName", launch.browser = T)"""
+      val launchCommand = "shiny::runExample(" + '"' + exampleName + '"' + ", launch.browser = T)"
       pressKeys(launchCommand)
       pressKeys(Keys.ENTER.toString)
```

Comment:
I was thinking instead of 2 `pressKeys` calls we could do something like:
```
val launchCommand = s"""shiny::runExample("$exampleName", launch.browser = T)${Keys.ENTER.toString}"""
pressKeys(launchCommand)
```
Not sure if that will help or not..